src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -3543,15 +3543,23 @@ ExplicitInterfaceSpecifierSyntax tryParseExplicitInterfaceSpecifier()
                 }
             }
 
-            // check for >>
+            // check for >> and >>>
             var opKind = opToken.Kind;
             var tk = this.CurrentToken;
-            if (opToken.Kind == SyntaxKind.GreaterThanToken && tk.Kind == SyntaxKind.GreaterThanToken)
+            if (opToken.Kind == SyntaxKind.GreaterThanToken && tk.Kind == SyntaxKind.GreaterThanToken &&
+                NoTriviaBetween(opToken, tk)) // no trailing trivia and no leading trivia
             {
-                // no trailing trivia and no leading trivia
-                if (NoTriviaBetween(opToken, tk))
+                var opToken2 = this.EatToken();
+                tk = this.CurrentToken;
+
+                if (tk.Kind == SyntaxKind.GreaterThanToken &&
+                    NoTriviaBetween(opToken2, tk)) // no trailing trivia and no leading trivia
+                {
+                    opToken2 = this.EatToken();
+                    opToken = SyntaxFactory.Token(opToken.GetLeadingTrivia(), SyntaxKind.GreaterThanGreaterThanGreaterThanToken, opToken2.GetTrailingTrivia());
```

Comment:
Should `MergeAdjacent` be used?

Reply:
> Should MergeAdjacent be used?

I am not inventing new ways of doing things, simply following what we do for ```SyntaxKind.GreaterThanGreaterThanToken```.